packages/scripts/entrypoint_nestri.sh

```
@@ -259,33 +258,38 @@
         fi
     done
 }
 
 main() {
-    load_envs
     parse_resolution "${RESOLUTION:-1920x1080}" || exit 1
+    get_container_info || {
+        log "Warning: Failed to detect container information."
+    }
+    # Make sure we have DBus
+    export $(dbus-launch)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)